output/schema/schema.json

```diff
@@ -121388,7 +121400,137 @@
           }
         }
       ],
-      "specLocation": "ml/_types/Anomaly.ts#L49-L64"
+      "specLocation": "ml/_types/Anomaly.ts#L53-L68"
+    },
+    {
+      "kind": "interface",
+      "name": {
+        "name": "AnomalyExplanation",
+        "namespace": "ml._types"
+      },
+      "properties": [
+        {
+          "description": "Impact from the duration and magnitude of the detected anomaly relative to the historical average.",
+          "name": "anomaly_characteristics_impact",
+          "required": false,
+          "type": {
+            "kind": "instance_of",
+            "type": {
+              "name": "integer",
+              "namespace": "_types"
+            }
+          }
+        },
+        {
+          "description": "Length of the detected anomaly in the number of buckets.",
+          "name": "anomaly_length",
+          "required": false,
+          "type": {
+            "kind": "instance_of",
+            "type": {
+              "name": "integer",
+              "namespace": "_types"
+            }
+          }
+        },
+        {
+          "description": "Type of the detected anomaly: `spike` or `dip`.",
+          "name": "anomaly_type",
+          "required": false,
+          "type": {
+            "kind": "instance_of",
+            "type": {
+              "name": "string",
+              "namespace": "_builtins"
+            }
+          }
+        },
+        {
+          "description": "Indicates reduction of anomaly score for the bucket with large confidence intervals. If a bucket has large confidence intervals, the score is reduced.",
+          "name": "high_variance_penalty",
+          "required": false,
+          "type": {
+            "kind": "instance_of",
+            "type": {
+              "name": "boolean",
+              "namespace": "_builtins"
+            }
+          }
+        },
+        {
+          "description": "If the bucket contains fewer samples than expected, the score is reduced. If the bucket contains fewer samples than expected, the score is reduced.",
```

Comment:
This is the same sentence twice.

Reply:
Thanks, fixed in c160143b1e5456dc4e0b9675e922ae7121177a35